docs/troubleshooting/attach_state_snapshot.md

```diff
@@ -0,0 +1,50 @@
+---
+id: attach-state-snapshot
+title: Attach State Snapshot
+sidebar_label: Attach State Snpashot
+description: Instructions on attaching a supporting state snapshot.
+---
+
+## Terminology {#terminology}
+State Snapshot is different from DB snapshot.
+State Snapshot is checkpoint of some columns of the full DB taken at the epoch boundary.
+It is used in state sync and resharding.
+
+State snapshots are identified by the last block hash of the epoch.
+We save state snapshot in `$NEAR_HOME_DATA/state_snapshot/$BLOCK_HASH`.
+We also save `$BLOCK_HASH` in DB to know which path to open when we need to use snapshot. 
+
+## How to attach state snapshot to existing node {#how to attach}
+1. Download state snapshot on your machine.
+You can download it to any directory, but `$NEAR_HOME_DATA/state_snapshot/$BLOCK_HASH` has to point to your new state snapshot.
+2. Create a support directory anywhere on the node. We will refer to it as `$OTHER_HOME`.
+3. Copy config to the new directory
+```bash
+cp $NEAR_HOME/config.json $OTHER_HOME/config.json
```

Comment:
I also copied genesis and node key. I'm not sure if both were required but at least one of them was.